.github/workflows/build-binder-image.yml

```diff
@@ -0,0 +1,12 @@
+name: Binder
+on: [push]
+
+jobs:
+  Create-MyBinderOrg-Cache:
+    runs-on: ubuntu-latest
+    steps:
+    - name: cache binder build on mybinder.org
+      uses: jupyterhub/repo2docker-action@dbb1a7b9b3bdddfacce83e156c11180a6811da93
```

Comment:
The repo [suggested](https://github.com/jupyterhub/repo2docker-action#cache-builds-on-mybinderorg) using `@master` but I figured better to pin to a tag

Reply:
Should we define a release branch where we run this action on so we don't update docker images every push?

Reply:
This matches the current binder setup. By default, binder will invalidate the cache any time the latest commit of the branch you point to has changed. By default the binder urls point to the `master` branch. The difference is now the docker image is pre-built instead of waiting 10 minutes when you want to load a notebook.